mix.exs

```diff
@@ -57,6 +57,7 @@ defmodule SurfaceSite.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
+      setup: ["deps.get", "ecto.setup", "cmd npm install --prefix assets"],
```

Comment:
I just realised we don't even have a database. Even the `SurfaceSite.Repo` entry is commented out in `application.ex`: https://github.com/surface-ui/surface_site/blob/ff91be2d53e0ce8629e353cb29397ec28a300f44/lib/surface_site/application.ex#L12

I wonder if we really need `ecto.setup`. Probably not, I guess.

Reply:
want me to remove as part of this PR?

Reply:
If you can confirm we don't need it, feel free to do it as part of this PR, otherwise, we can check that later.

Reply:
done. Ecto remains as a dependency for one of the examples